src/spryker/application/skeleton/composer-harness.json.twig

```diff
@@ -59,11 +59,18 @@
     "test-acceptance": [
       "true"
     ],
-    "compatibility": "bin/phpcs -s --standard=vendor/phpcompatibility/php-compatibility/PHPCompatibility/ --runtime-set testVersion 7.4-"
+    "compatibility": "bin/phpcs -s --standard=vendor/phpcompatibility/php-compatibility/PHPCompatibility/ --runtime-set testVersion {{ @('php.version') }}-",
+    "phpstan": [
+      "php -d memory_limit=1024M vendor/bin/phpstan analyze -c phpstan.neon src/ -l 5"
+    ]
   },
   "config": {
     "platform": {
-      "php": "7.4.10"
+      "php": "{{ @('php.version') }}"
```

Comment:
just a note here, that if php.version is 7.4, that will likely be taken to mean 7.4.0, and composer reqs that do things like > 7.4.3 will fail